README.md

```diff
@@ -27,12 +43,32 @@ Ask us questions, talk about contributing, or just say hi on [Slack](https://vsc
 
 Donations help convince me to work on this project rather than my other (non-open-source) projects. I'd love to work on VSCodeVim full time, but I need money to live!
 
+## Contents
 
-## Configuring VSCodeVim
+* [Quick example settings](#quick-example-settings)
```

Comment:
Yea good idea, I knew this section was 'off'. I'll move the quick settings into Settings, and rename this to 'Getting started', and might even include some links to some vim learning resources?